motor_num: 30
actions_size: 30
dt: 0.0025
xsense_data_roll_offset: 0.0
zero_pos_offset:
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
- 0.0
ct_scale:
- 2.16
- 2.16
- 2.16
- 3.207
- 3.207
- 3.37
- 2.43
- 2.43
- 1.71
- 2.16
- 2.16
- 3.207
- 3.37
- 2.43
- 2.43
- 1.71
- 2.16
- 2.16
- 2.3
- 2.0
- 2.3
- 2.0
- 2.6
- 2.6
- 2.3
- 2.0
- 2.3
- 2.0
- 2.6
- 2.6
joint_kp_p:
- 10.0
- 10.0
- 10.0
- 20.0
- 60.0
- 20.0
- 10.0
- 10.0
- 15.0
- 8.0
- 8.0
- 60.0
- 20.0
- 10.0
- 10.0
- 15.0
- 8.0
- 8.0
- 700.0
- 700.0
- 500.0
- 700.0
- 30.0
- 16.8
- 700.0
- 700.0
- 500.0
- 700.0
- 30.0
- 16.8
joint_kd_p:
- 0.5
- 0.5
- 0.5
- 1.0
- 3.0
- 1.5
- 1.0
- 1.0
- 1.0
- 0.5
- 0.5
- 3.0
- 1.5
- 1.0
- 1.0
- 1.0
- 0.5
- 0.5
- 10.0
- 10.0
- 5.0
- 10.0
- 2.5
- 1.4
- 10.0
- 10.0
- 5.0
- 10.0
- 2.5
- 1.4
joystick:
  forward_command_offset: 0.0
  lateral_command_offset: 0.0
  rotation_command_offset: 0.0
simulation: false
mlp:
  path: /config/policy/policy
